storage/innobase/log/log0log.cc

```diff
@@ -2896,6 +2895,7 @@ log_archive_all(void)
 
 	mutex_exit(&(log_sys->mutex));
 
+	my_sd_extend_timeout(INNODB_EXTEND_TIMEOUT_INTERVAL);
```

Comment:
Log archiving is dead code inside InnoDB, and not much tested in XtraDB. If you really want to extend the timeouts here, then `log_archive_do()` would be the proper place.

My understanding is that the log archiving is executed while the server is running normally. It is not a one-time startup or shutdown task, and therefore probably should not be instrumented in this patch.

Reply:
ok. omitted.